test/test-rotateElement.js

```diff
@@ -0,0 +1,902 @@
+    })
+
+    it("the item returns to its original position", function(){
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            -----
+            ---X-
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+		let expectedResult = tetris.getCoordinates(`
+            -----
+            ----X
+            -----
+            -----
+            -----
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+describe("element rotation around the axis 5x5 not on diagonal", function () {
+
+    it("element move the top", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            ----X
+            -----
+            -----
+            -----
+            
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+        let expectedResult = tetris.getCoordinates(`
+            -X---
+            -----
+            -----
+            -----
+            -----
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+    it("element move the left", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -X---
+            -----
+            -----
+            -----
+            -----
+            
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+        let expectedResult = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            X----
+            -----
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+    it("element move the right", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            X----
+            -----
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+        let expectedResult = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            -----
+            ---X-
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+    it("the item returns to its original position", function(){
```

Comment:
these test names are meaningless

Reply:
Please rename all test cases so that:
1) they reflect exactly what is going on
2) they don't repeat across test suite (describe ...) - test names should be unique in entire file